pom.xml

```diff
@@ -41,6 +41,10 @@
 			<groupId>org.scijava</groupId>
 			<artifactId>ui-behaviour</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>mpicbg</groupId>
+			<artifactId>mpicbg</artifactId>
```

Comment:
Perhaps add a (minimum) version to make it more explicit, which version is used.

Reply:
No that's not how it works. The version numbers are listed in the parent pom.

Reply:
Why is it then actually added at all to the pom.xml of the plugin? Isn't it then already imported as a dependency via the parent pom?

If not, I would the expect re-using the version number defined in the parent pom:

![grafik](https://user-images.githubusercontent.com/10515534/224696838-b9bef83e-e147-4537-92f5-2a89d7d4c683.png)

Reply:
Trust me I did it the correct way. Look for example at the main mastodon or fiji repository:
* https://github.com/mastodon-sc/mastodon/blob/master/pom.xml
* https://github.com/fiji/fiji/blob/master/pom.xml

This PR is not the appropriate place to discuss how maven works.